Blog/models.py

```diff
@@ -6,14 +6,17 @@
 
 
 # inject get_absolute_url method to `User` class by Django
+from vote.models import VoteModel
```

Comment:
This should be moved to the import section of the file, not right after the comment for  `get_absolute_url()`.